mem0/configs/base.py

```diff
@@ -36,4 +36,13 @@ class MemoryConfig(BaseModel):
     history_db_path: str = Field(
         description="Path to the history database",
         default=os.path.join(mem0_dir, "history.db"),
-    )
\ No newline at end of file
+    )
+    limit: int = Field(
+        description="The maximum number of memories to return",
+        default=10,
```

Comment:
Hey, can you please set default to `5`.

Reply:
@vsokolovskii I think it would be better to add fields "limit" and "similarity_threshold" not as a separated parameters in a generalt config, but add them into the vector_store config parameters

Reply:
Yes @max-rtmlab It's a good idea. @vsokolovskii Can you please make those changes?

Reply:
the vector_store config paramters are being used by VectorStoreFactory to create the instance of the vector store class so these params do not belong there since they are used during the search and not during the VDB client initialization. I think we could wrap them to sth like 'Search Configuration' but it will not be the part of the vector_store

Reply:
Does it make sense? Do we want to leave it like this or you want to wrap them to Search Confuration param?

Reply:
+ will add the tests and update the documentation soon

Reply:
Good work @vsokolovskii

Reply:
Also @vsokolovskii Please fix the merge conflicts.